.github/workflows/on-release-manual-dispatch.yml

```diff
@@ -0,0 +1,50 @@
+name: Release Dispatch
```

Comment:
This file is a copy of https://github.com/pulumi/pulumi/blob/master/.github/workflows/on-release.yml, but setup to allow manual trigger with input variables, rather than being triggered by a release.